src/players.ts

```diff
@@ -122,7 +123,7 @@ export class FishPlayer {
 	showRankPrefix:boolean;
 
 	constructor({
-		uuid, name, muted = false, autoflagged = false, unmarkTime: unmarked = -1,
+		uuid, name, muted = false, autoflagged = false, vpn = false, unmarkTime: unmarked = -1,
```

Comment:
You're adding this under stored data, but haven't added it to the serialization code. I think you meant to add it as a transient, which gets wiped on server restart?